### PR TITLE
Simplify initial application setup

### PR DESCRIPTION
**Pull request by scytacki**
This removes is the usage of 'setup.rb' to setup a new portal instance. 
Instead the portal a few files are copied and the standard rails tasks `db:setup` is run.
The documentation is updated and simplified.  You can see the basic steps here:
https://github.com/concord-consortium/rigse/blob/simplify-db-initialization/README.md#simple-getting-started

This does remove some of the complex setup options that we had previously. And it means an NCES school importing has to be done more manually. However the normal developer shouldn't need to work about NCES schools, and hopefully that whole process will be extracted into its own repository in the near future.

@knowuh or @psndcsrv  when you have a chance please take a look.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,78 +8,29 @@ Setup
...
-    echo "rvm use 1.9.3-p125@xproject" > .rvmrc
-
-Test out the `.rvmrc` file and start using the gemset
-
-    cd ..;cd xproject
+    bundle install
+    cp config/database.sample.yml config/database.yml (need to fix the mysql password and/or user)
+    cp config/settings.sample.yml config/settings.yml
+    cp config/app_environment_variables.sample.rb config/app_environment_variables.rb
+    cp config/sis_import_data.sample.yml config/sis_import_data.yml
```
Comment (knowuh) [inline README.md]: We are removing sis_import in https://github.com/concord-consortium/rigse/pull/92  -- so we can probably delete this copy.

**Comment by knowuh** [comment]
Beautiful -- Love all that red.

**Comment by psndcsrv** [comment]
Looks good! I agree with @knowuh, all the red is nice to see.

**Comment by scytacki** [comment]
I'll remove the sis copy in the sis pull request. Without the SIS copy there something that fails, I can't remember what.

**Pull request merged**